l10n_th_withholding_tax_report/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+To use this module, you need to:
```

Comment:
To run Withholding Tax reports,
- Go to menu Invoicing > Reporting > Withholding Reports
- Select Income Tax Form and Date Range
- You can choose to View HTML, Export PDF and Export Excel